### DISCUSSION TITLE
VS autocompletion showing keyword "string" as first candidate in namespace scope

### DISCUSSION THREAD

**Discussion by acaly** [dotnet/roslyn#66948] [General] [answered]
Sorry I am not sure whether this is the appropriate place to report this. Whenever I wrote `public readonly str`, it shows `string` as the first candidate. This is annoying, because I have to type in 4 letters to get the correct `struct`. I think in namespace scope, types should not be shown after `readonly`.

**Comment by RikkiGibson** [comment, marked as answer]
Please reopen this as an issue. At a glance it seems like this is an opportunity for improvement, but we'll want to confirm with the IDE team.

**Reply by acaly** [reply]
Thanks! Opened as #67061.
